src/main/java/ch/uzh/ifi/hase/soprafs22/controller/InGameWebsocketController.java

```diff
@@ -15,18 +17,26 @@
 @Controller
 public class InGameWebsocketController {
 
+    private final InGameWebsocketService service;
+
+    InGameWebsocketController(InGameWebsocketService service) {
+        this.service = service;
+    }
+    
     @MessageMapping("/move")
     @SendTo("/client/move")
-    public ExampleMoveGetDTO exampleMove(ExampleMovePostDTO exampleMovePostDTO, Principal principal) throws Exception {
+    public MoveGetDTO exampleMove(MovePostDTO exampleMovePostDTO, Principal principal, Long gameId) throws Exception {
```

Comment:
Did you test it with the principal in the middle? Not sure whether that works or not 🤔 Better be safe than sorry 😉 

```suggestion
    public MoveGetDTO exampleMove(MovePostDTO exampleMovePostDTO, Long gameId, Principal principal) throws Exception {
```

Reply:
Does the order of the arguments have an influence on something?

Reply:
Not really but since exampleMovePostDTO and gameId have to be sent as a JSON Object and the Principal is generated by dependency injection (with the Auth headers), it's best practice to have them in this order.

I am not sure if it has an influence in Java Spring. But it does have an influence in PHP Symfony for example.